src/sas/sascalc/realspace/VolumeCanvas.py

```diff
@@ -409,38 +409,37 @@ def getParam(self, name):
         toks = name.split('.')
         if len(toks) == 1:
             try:
-                self.params.has_key(toks[0])

```

Comment:
I don't think`has_key` would ever raise a `KeyError`. If the key isn't in the dict, then it returns False, otherwise it returns True.

```
>>> a = {'a':0, 'b':1, 'c':2}
>>> a.has_key('a')
True
>>> a.has_key([])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'list'
>>> a.has_key('s')
False
>>> 
```